README.md

```diff
@@ -19,6 +19,33 @@ and to keep them as similar as possible:
 See [fedora-coreos-pipeline](https://github.com/coreos/fedora-coreos-pipeline) as an
 example pipeline.
 
+### Build Process
+
+```
++-------------------------------------------+            +----------------------------------+  
+|                                           |            |   CoreOs Assembler Repo          |                 
+|         FCOS Config                       |            |                                  |
+|                                           |            |   * Build Script                 |
+|    * rpm-ostree treefile                  |            |   * Dockerfile                   |
+|    * Kickstart                            |            |                                  |
```

Comment:
I think we removed the kickstart from the config - so probably can remove that.

Reply:
It's currently part of the build scripts.

Reply:
but it's definitely not part of the FCOS config github repo

Reply:
@dustymabe @ajeddeloh: should I remove the kickstart from the FOCS config?

Reply:
I would

Reply:
@dustymabe: sure, let me remove it.